syntax = "proto2";

message GoYoData {
    required int32 x_accel = 1;
    required int32 y_accel = 2;
    required int32 z_accel = 3;
    required int32 fsr = 4;
}
